### DISCUSSION TITLE
Github code indexing is broken

### DISCUSSION THREAD

**Discussion by po5** [tomasklaen/uosc#757] [Q&A]
The main code now being in a `dist` folder, it renders Github search unusable. 

![image](https://github.com/tomasklaen/uosc/assets/42466980/f5fcdba0-f63c-4814-836c-bf94141e3928)

The only Lua files it still finds are tests: https://github.com/search?q=repo%3Atomasklaen%2Fuosc++language%3ALua&type=code

This makes it more annoying to dig through the code without cloning the repo. Using github.dev for in-browser vscode works I guess.

**Comment by tomasklaen** [comment]
:( All right, I'll move it to `src` and update all scripts and tools.

**Comment by po5** [comment]
Thinking about it some more it may be because there's a `src` folder present, renaming it to `tools_src` or something like that may fix it.

**Comment by tomasklaen** [comment]
Done. Hopefully this is the last time I had to restructure the repo :)
